<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rencontre_pokemon_sauvage</class>
 <widget class="QDialog" name="rencontre_pokemon_sauvage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fond_prairie">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/prairie_et_ciel.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="pokemon_sauvage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/images_pokemon/pokemons_finaux/face/Diglett.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="perso_qui_court">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>-30</y>
     <width>181</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/perso_attaque.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_fuir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>fuir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_combattre">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>COMBATTRE !</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Un nom_pokémon sauvage apparaît !</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
